src/globals/scss/styles.scss

```diff
@@ -132,8 +132,7 @@ $deprecations--message: 'Deprecated code was found, this code will be removed be
 @if (length($deprecations--reasons) > 0) {
   $deprecations--message: '';
   @each $reason in $deprecations--reasons {
-    $deprecations--message: '#{$deprecations--message}
-REASON: #{$reason}';
+    $deprecations--message: '#{$deprecations--message} REASON: #{$reason}';
```

Comment:
Is there anyway to add a newline or no? 🤔

Reply:
We didn't add one [here](https://github.com/carbon-design-system/carbon/pull/2905/files#diff-b22daf8ba1fef227c061cf5feeefc09dR169), so I figured it wasn't possible when I reviewed